src/vmm/src/guest_config/x86_64/mod.rs

```diff
@@ -0,0 +1,16 @@
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::collections::HashMap;
+
+use crate::guest_config::cpuid::Cpuid;
+
+/// CPU configuration for x86_64 CPUs
+pub struct X86_64CpuConfiguration {
+    /// CPUID configuration
+    pub cpuid: Cpuid,
+    /// Register values as a key pair for model specific registers
+    /// Key: MSR address
+    /// Value: MSR value
+    pub msrs: HashMap<u64, u64>,
```

Comment:
The input address space of MSR 32-bit wide.

Reply:
You are correct. For the feature branch this isn't a blocker, this can be fixed downstream.